package/crossplane.yaml

```diff
@@ -1,14 +1,16 @@
 apiVersion: meta.pkg.crossplane.io/v1alpha1
 kind: Provider
 metadata:
-  name: provider-azure
+  name: provider-influxdb
   annotations:
     meta.crossplane.io/maintainer: Crossplane Maintainers <info@crossplane.io>
-    meta.crossplane.io/source: github.com/crossplane/provider-azure
+    meta.crossplane.io/source: github.com/crossplane-contrib/provider-influxdb
     meta.crossplane.io/license: Apache-2.0
     friendly-name.meta.crossplane.io: Provider InfluxDB
     friendly-group-name.meta.crossplane.io/influxdb.crossplane.io: InfluxDB
-    friendly-kind-name.meta.crossplane.io/organizations.azure.crossplane.io: Organization
+    friendly-kind-name.meta.crossplane.io/organizations.influxdb.crossplane.io: Organization
```

Comment:
What exactly are these friendly name annotations used for?

Reply:
They are standard package annotations that help UIs that need human-friendly names. See https://github.com/crossplane/crossplane/blob/master/design/one-pager-package-annotations.md for more details.

Reply:
Thanks @muvaf. My understanding is that it's part of a goal to encourage a consistent set of annotations across providers.